src/interpret.ml

```diff
@@ -655,13 +655,12 @@ and interpret_stmt stmt env =
       env )
 
 and interpret_block b env =
-  try List.fold_left (fun e stmt -> interpret_stmt stmt e) env b
-  with Return_catch (el, so, env) -> raise (Return_catch (el, so, env))
```

Comment:
Looks like this `try .. with` was useless ?

Reply:
ah, ok ;)